encorelab/SCORE-Client
# Sync with WISE-Client/develop Sept. 2023

## Changes
- Syncs SCORE-Client with WISE-Client/develop, including myChoiceChosen token and archive run feature

## Test
- Test with https://github.com/encorelab/SCORE-API/pull/18
- Make sure everything work as before
- New features
   - myChoiceChosen token
   - isLowestWorkgroupIdInPeerGroup token
   - click question bank item to add to peer chat
   - specify different label for "Question Bank" (e.g. "Prompt Bank")
